sklearn/ensemble/_hist_gradient_boosting/gradient_boosting.py

```diff
@@ -1577,11 +1577,16 @@ class HistGradientBoostingRegressor(RegressorMixin, BaseHistGradientBoosting):
         .. versionadded:: 0.23
 
     scoring : str or callable or None, default='loss'
-        Scoring parameter to use for early stopping. It can be a single
-        string (see :ref:`scoring_parameter`) or a callable (see
-        :ref:`scoring_callable`). If None, the estimator's default scorer is used. If
-        ``scoring='loss'``, early stopping is checked w.r.t the loss value.
-        Only used if early stopping is performed.
+        Scoring method to use for early stopping. Only used if early stopping
+        is performed. Options:
```

Comment:
I have the feeling that those 2 phrases were a bit redundant, with this tweak at least there's a clearer difference
```suggestion
        Scoring method to use for early stopping. Only used if `early_stopping`
        is enabled. Options:
```